### PR TITLE
Feature/admin comments validation

### PR DESCRIPTION
**Pull request by fabianrupin195**
#23

### CODE REVIEW

File: src/Blog/Backend/Controller/CommentsController.php [hunk 1]
```diff
@@ -49,20 +50,40 @@ public function __construct(
...
 
+        if ($request->getMethod() === 'POST') {
+            $commentId = $request->request->get('id');
+            $newStatus = $request->request->get('status');
+            $authorID = $this->adminRepository->findAuthorByCommentId($commentId);
+
+            if ($authorID === $user->id or $user->getRole() === 'superadmin') {
+                $isQuerySucess = $this->adminRepository->modifyStatusComment($commentId, $newStatus);
+                if ($isQuerySucess) {
+                    $message = 'Vous avez passer le commentaire '.$commentId.' à '.$newStatus;
```
Comment (juuuuuu) [inline src/Blog/Backend/Controller/CommentsController.php]: `passé`

File: src/Blog/Backend/views/showComments.twig [hunk 2]
```diff
@@ -48,6 +56,22 @@
                         </tfoot>
                         <tbody>
                         {% for comment in comments %}
+
+                            {% if (comment.validation_status == 'pending') %}
+                                {% set statusOption = [
+                                    {'value' : 'validate', 'texte': 'Valider'},
```
Comment (juuuuuu) [inline src/Blog/Backend/views/showComments.twig]: `texte` is a french word.

File: src/Blog/Frontend/Controller/ErrorController.php [hunk 3]
```diff
@@ -10,7 +10,7 @@ class ErrorController
     public function exceptionAction(FlattenException $exception): Response
     {
         // Sometimes the $msg is not explicit
-        //var_dump($exception);
+        var_dump($exception);
```
Comment (juuuuuu) [inline src/Blog/Frontend/Controller/ErrorController.php]: I guess you forgot to remove this debug line?

File: web/index.php [hunk 4]
```diff
@@ -10,6 +10,7 @@
 include __DIR__ . '/../src/container.php';
 
 $request = Request::createFromGlobals();
+$request->query->all();
```
Comment (juuuuuu) [inline web/index.php]: `all()` method returns an array, but your are not affecting any variable to this method call.

**Comment by fabianrupin195** [comment]
c'est corrigé et mergé !

**Pull request merged**